ledger/acctupdates_test.go

```diff
@@ -597,82 +598,6 @@ func TestAcctUpdates(t *testing.T) {
 	}
 }
 
-// TestAcctUpdatesFastUpdates tests catchpoint label writing datarace
-func TestAcctUpdatesFastUpdates(t *testing.T) {
```

Comment:
Is this test being removed? I don't see it being added back in by-name anywhere. Could you include mention of why that's happening in the PR?

Reply:
it is moved, and renamed since it is not account updates anymore

Reply:
got it, thanks.